### DISCUSSION TITLE
Running queries for issues by character

### DISCUSSION THREAD

**Discussion by CountZeroOr** [bpepple/metron#120] [Q&A] [answered]
I've been thinking about putting together a project somewhat like what Travis Starnes has done in the past with his Complete Marvel Reading Order (https://cmro.travis-starnes.com/), except for, for example, DC, using the data from Metron to help get things set up, as there's some functionality that is not in Metron that I need for what I'm trying to do (marking comics as read, adding links to where you can read the comics on the DC Universe site as they become available, etc.).

To make this work what I would like to be able to do is pull a list of issues by character. I'm currently unable to find any documentation on how to do this in the API. I know it can be done through the web interface - here's a list of relevant issues for, for example, Batman - https://metron.cloud/character/batman/issue_list/

So, what I'd like to know - can I generate this separately through the API, or would I need to pull together individual queries for DC Comics Characters, DC Comics Volumes, and then DC Comics Issues, and then individual issues, in order to put together a list of "List of issues the Christopher Chance (Human Target) appears in" (for example)?

**Comment by bpepple** [comment]
I've been meaning to expose an API issue list endpoint for character & teams, just hadn't gotten around to it. It's pretty easy thing to expose and I can probably implement it either on Saturday or Sunday.

The endpoint will be something like this:

/api/character/{id}/issue_list/

**Comment by bpepple** [comment, marked as answer]
Ok, I've added issue_list endpoints for Characters & Teams in #121. The docs should be automatically updated for this change. If you are using python for your project, my wrapper for Metrons' API should have a new release supporting this in an hour or so. If you have any additional questions, don't hesitate to ask.
